### ISSUE TITLE
[BUG] Modal Buttons missing accessibility attributes

### ISSUE THREAD

**Issue by hrgondaliya** [trimble-oss/modus-web-components#1020]
**Describe the bug**
Modal Button (primary-button and secondary-button) are missing accessibility attributes

**To Reproduce**

Go to https://modus-web-components.trimble.com/?path=/docs/components-modal--default
Click on 'open modal'
The Close button should have attributes role="button" and aria-label="Close"

1. Go to https://modus-web-components.trimble.com/?path=/docs/components-modal--default
2. Click on 'open modal'
3. Scroll down to the popup model buttons (primary-button ("Sweet!") and secondary-button ("Save changes"))
4. Both buttons are missing aria-label

**Expected behavior**
The modus-modal's both buttons should have the aria-label property for the accessibility

**Screenshots**
![image](https://user-images.githubusercontent.com/49644189/220041635-1a2e6c18-d307-4884-8051-68ec76b8691a.png)

**Comment by coliff** [comment]
Good spot! Look forward to seeing your PR. Also I think the `Sweet!` and `Save Changes` buttons could have `type="button"` (and the `role="button"` can be removed).
